integrations/facebook-pixel/lib/index.js

```diff
@@ -514,20 +500,42 @@ FacebookPixel.prototype.checkoutStarted = function(track) {
 };
 
 /**
- * mappedContentTypesOrDefault returns an array of mapped content types for
- * the category - or returns the defaul value.
- * @param {Facade.Track} track
- * @param {Array} def
+ * Returns an array of mapped content types for the category,
+ * the provided value as an integration option or the default provided value.
+ *
+ * @param {Facade.Track} track Track payload
+ * @param {Array} defaultValue Default array value returned if the previous parameters are not defined.
+ *
+ * @return Content Type array as defined in:
+ * - https://developers.facebook.com/docs/facebook-pixel/reference/#object-properties
+ * - https://developers.facebook.com/docs/marketing-api/dynamic-ads-for-real-estate/audience
  */
-FacebookPixel.prototype.mappedContentTypesOrDefault = function(category, def) {
-  if (!category) return def;
+FacebookPixel.prototype.getContentType = function(track, defaultValue) {
+  // 1- Integration options takes preference over everything
+  var options = track.options('Facebook Pixel');
```

Comment:
Does this code get transpiled? Any reason we use `var` instead of `const`/`let`?

Reply:
It doesn't. `const` and `let` are not supported in old browsers.

Reply:
Got it yep.